### ISSUE TITLE
Little unsure how to get this working initially

### ISSUE THREAD

**Issue by twhite96** [sequelize/sequelize-auto#156]
I am not sure if I need a specific database file or a js file. 

Every time I run the command, it tells me `Users.js does not exist` yet it *does* exist. 

I have these options in my `Users.js` file:

```javascript
var auto = new SequelizeAuto('database', 'user', 'pass', {
    host: 'localhost',
    dialect: 'postgres',
    port: 'port',
    directory: '/models',
    additional: {
        timestamps: false
    },
    tables: ['Users', 'Texts']
})
```

I am getting these errors:

```sh
{ SequelizeBaseError: database "Users.js" does not exist
    at /usr/local/lib/node_modules/sequelize-auto/node_modules/sequelize/lib/dialects/postgres/connection-manager.js:110:20
    at Connection.<anonymous> (/usr/local/lib/node_modules/pg/lib/client.js:186:5)
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:191:7)
    at Socket.<anonymous> (/usr/local/lib/node_modules/pg/lib/connection.js:136:12)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at TCP.onread (net.js:560:20)
  name: 'SequelizeConnectionError',
  message: 'database "Users.js" does not exist',
  parent: 
   { error: database "Users.js" does not exist
       at Connection.parseE (/usr/local/lib/node_modules/pg/lib/connection.js:569:11)
       at Connection.parseMessage (/usr/local/lib/node_modules/pg/lib/connection.js:396:17)
       at Socket.<anonymous> (/usr/local/lib/node_modules/pg/lib/connection.js:132:22)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:191:7)
       at readableAddChunk (_stream_readable.js:178:18)
       at Socket.Readable.push (_stream_readable.js:136:10)
       at TCP.onread (net.js:560:20)
     name: 'error',
     length: 93,
     severity: 'FATAL',
     code: '3D000',
     detail: undefined,
     hint: undefined,
     position: undefined,
     internalPosition: undefined,
     internalQuery: undefined,
     where: undefined,
     schema: undefined,
     table: undefined,
     column: undefined,
     dataType: undefined,
     constraint: undefined,
     file: 'postinit.c',
     line: '841',
     routine: 'InitPostgres' },
  original: 
   { error: database "Users.js" does not exist
       at Connection.parseE (/usr/local/lib/node_modules/pg/lib/connection.js:569:11)
       at Connection.parseMessage (/usr/local/lib/node_modules/pg/lib/connection.js:396:17)
       at Socket.<anonymous> (/usr/local/lib/node_modules/pg/lib/connection.js:132:22)
       at emitOne (events.js:96:13)
       at Socket.emit (events.js:191:7)
       at readableAddChunk (_stream_readable.js:178:18)
       at Socket.Readable.push (_stream_readable.js:136:10)
       at TCP.onread (net.js:560:20)
     name: 'error',
     length: 93,
     severity: 'FATAL',
     code: '3D000',
     detail: undefined,
     hint: undefined,
     position: undefined,
     internalPosition: undefined,
     internalQuery: undefined,
     where: undefined,
     schema: undefined,
     table: undefined,
     column: undefined,
     dataType: undefined,
     constraint: undefined,
     file: 'postinit.c',
     line: '841',
     routine: 'InitPostgres' } }
```

Not sure what I am doing wrong.

**Comment by durango** [comment]
Postgres is case-sensitive is your table name called `Users` or `users`?

**Comment by twhite96** [comment]
@durango Users

**Comment by durango** [comment]
Can you copy & paste the schema so that I may test locally please?

**Comment by twhite96** [comment]
Still working on this. I am using Heroku and didn't have a schema, per se, locally. Anyway I can use this with Heroku? Anywhere I can find out how?

**Comment by manjeet13** [comment]
Is there a way to specify location where the model will be generated doing this way?

**Comment by steveschmitt** [comment]
I think this is fixed in `sequelize-auto@0.5.0`
Please try it and close the issue if so.  Thanks!
